vendor/ember-let/register.js

```diff
@@ -0,0 +1,70 @@
+/* global require */
+(function() {
+
+  /** start of babel output */
+  var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+  function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+  function _possibleConstructorReturn(self, call) { if (!self) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return call && (typeof call === "object" || typeof call === "function") ? call : self; }
+
+  function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
+  /** end babel output */
+
+  var Ember = require('ember').default;
+
+  if (Ember.__loader.registry['ember-glimmer/syntax']) {
+    var registerSyntax = Ember.__loader.require('ember-glimmer').registerSyntax;
+    var StatementSyntax = Ember.__loader.require('glimmer-runtime').StatementSyntax;
+
+    var LetSyntax = function (_StatementSyntax) {
```

Comment:
Lets use plain ole ES5 here since we can't get pretty transpiled output.

Something like:

```js
var registerSyntax = Ember.__loader.require('ember-glimmer').registerSyntax;
var StatementSyntax = Ember.__loader.require('glimmer-runtime').StatementSyntax;

function LetSyntax(args, templates) {
  StatementSyntax.call(this);

  this.args = args;
  this.templates = templates;
}

LetSyntax.prototype.compile = function LetSyntax_compile(dsl) {
  dsl.block({ templates: this.templates, args: this.args }, function(dsl) {
    dsl.evaluate('default');
  });
};

LetSyntax.create = function(environment, args, templates) {
  return new this(args, templates);
};

registerSyntax('let', LetSyntax);
```